tutorials/cloud-functions-async/index.md

```diff
@@ -88,12 +89,15 @@ the task, it moves the job from `Created` to `Running`. When the task is complet
 
 ## Setup
 
-1.  Create a project in the [GCP Console][console].
-1.  [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project)
-1.  Use [Cloud Shell][shell] or install the [Google Cloud SDK][sdk].
-1.  Enable Cloud Functions:
+1. Create a project in the [GCP Console][console].
+1. [Enable billing for your project](https://cloud.google.com/billing/docs/how-to/modify-project).
+1. Use [Cloud Shell][shell] or install the [Google Cloud SDK][sdk].
+1. Enable Cloud Functions, Firestore, and Pub/Sub APIs:
 
-        gcloud services enable cloudfunctions.googleapis.com
+        gcloud services enable cloudfunctions.googleapis.com firestore.googleapis.com pubsub.googleapis.com
+	    
+1. Install [`jq`][jq], [`curl`][curl], and [`watch`][watch]. These tools are already in Cloud Shell,
```

Comment:
jq may be alt/better than python -m json.tool, but if you are suggesting we switch to that, we need to change the usage of watch | as well right?

Reply:
You already use `jq` in [Create a job](https://github.com/GoogleCloudPlatform/community/blob/dcc67dc763ed208a0a128d4151bf903cb405f408/tutorials/cloud-functions-async/index.md#create-a-job). I was just documenting it 😄

Reply:
Either we need to remove `jq` from the Create a job or add it to watch and remove the python dep. I vote for removing the python dep since jq is actually installed standard on all Google things (cloud shell, etc), and it comes bundled in many distros now.

Reply:
ah  - I had completely missed / forgotten this - yeah, I'm good with consolidating on jq - thx

Reply:
PR on your PR so we just do this on one place :)